contrib/docker/README.md

```diff
@@ -0,0 +1,39 @@
+# Getting Start with Superset using Docker
```

Comment:
CI is complaining that ASF license is missing from this file

Reply:
Thanks, fixed.